boards/v2/qemu/qemu_leon3/Kconfig.qemu_leon3

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_QEMU_LEON3
-	bool "QEMU LEON3 target"
-	depends on SOC_LEON3
+	bool
+	select SOC_LEON3
 	select QEMU_TARGET
 	select CPU_HAS_FPU
```

Comment:
those are Zephyr specifics and not re-usable when sourcing Kconfig tree outside, for example in sysbuild.

Should go to the regular `[boards/v2/qemu/qemu_leon3/Kconfig` file.

Reply:
Fixed

Reply:
Not fixed.